subprojects/core/src/integTest/groovy/org/gradle/api/InitScriptExecutionIntegrationTest.groovy

```diff
@@ -271,6 +271,15 @@ initscript {
         output.contains("Project 'root'")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/17555")
+    def "init script file is a dotfile"() {
+        def initScript = file('.empty')
+        initScript << 'println "greetings from user home"'
+        expect:
+        executer.withArguments('--init-script', initScript.absolutePath)
+        succeeds()
```

Comment:
```suggestion
        succeeds()
        
        then:
        output.contains("greetings from user home")
```

Reply:
Thanks, I have changed it manually for convenient